cases/util/loggableFetcher.js

```diff
@@ -1,8 +1,13 @@
 import { fetch } from "./fetchShim";
 
-export const loggableFetch = async (requestURL, requestDictionary) => {
+export const loggableFetch = async (
+  requestURL,
+  requestDictionary = {},
+  responseIsJSON = true,
+) => {
   const response = await fetch(requestURL, requestDictionary);
-  const json = await response.json();
+
+  const json = responseIsJSON ? await response.json() : await response.text();
```

Comment:
This is not great, as the response body may not be a JSON.

Reply:
i think its fine to just not use this for the toml tests, they're pretty simple and we don't need any complex inspection for it do we?

Reply:
Yeah, I guess you're right 😢